src/stores/apps.store.ts

```diff
@@ -77,25 +77,49 @@ type App = {
   }
 }
 
+type AppOverview = {
+  id: AppId
+  name: string
+  bandwidth: {
+    allowed: number
+    consumed: number
+  }
+  storage: {
+    allowed: number
+    consumed: number
+  }
+  estimatedCost: number
+  noOfFiles: number
+  totalUsers: number
+  createdAt: string
+}
+
 type AppState = {
   appIds: AppId[]
   appsById: {
     [key: AppId]: App
   }
+  appsOverview: {
```

Comment:
Maybe we should all this `appsOverviewById` to keep the same convention as `appsById`?

Reply:
Resolved at https://github.com/arcana-network/developer-dashboard/pull/135/commits/9e785654fea8c963d6010cff341b8ceaa7fe2278